src/sentry/tasks/post_process.py

```diff
@@ -687,7 +716,7 @@ def process_event(data: dict, group_id: Optional[int]) -> Event:
 
     # Re-bind node data to avoid renormalization. We only want to
     # renormalize when loading old data from the database.
-    event.data = EventDict(event.data, skip_renormalization=True)
+    event.data = EventDict(event.data, skip_renormalization=True)  # type: ignore  # NodeData uses EventDict as wrapper
```

Comment:
should probably fix the typing so this is allowed -- also `# type: ignore` should always have a code like `# type: ignore[code-here]`

Reply:
@asottile-sentry the error here is an assignment one that occurs because `data` has type `NodeData` but is receiving `EventDict`. If you go to `src/sentry/eventstore/models.py`, the setter for `data` initializes a `NodeData` instance with `EventDict` as the wrapper, hence why this operation works, but I'm not sure how to best get mypy to recognize this